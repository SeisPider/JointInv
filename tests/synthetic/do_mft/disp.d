SURF96 R U T  0          95       3.959      0.5224 
SURF96 R U T  0          90       3.948      0.4921 
SURF96 R U T  0          85       3.932      0.4611 
SURF96 R U T  0          80       3.917      0.4307 
SURF96 R U T  0          75       3.896      0.3995 
SURF96 R U T  0          70       3.874      0.3686 
SURF96 R U T  0          65       3.844       0.337 
SURF96 R U T  0          60       3.801      0.3041 
SURF96 R U T  0          55       3.744      0.2705 
SURF96 R U T  0          50       3.665      0.2356 
SURF96 R U T  0          48       3.626      0.2215 
SURF96 R U T  0          46       3.579      0.2068 
SURF96 R U T  0          44       3.515      0.1908 
SURF96 R U T  0          42       3.453      0.1757 
SURF96 R U T  0          40       3.374      0.1598 
SURF96 R U T  0          38       3.296      0.1448 
SURF96 R U T  0          36       3.202      0.1295 
SURF96 R U T  0          34       3.104       0.115 
SURF96 R U T  0          32       3.003      0.1013 
SURF96 R U T  0          30       2.923     0.08994 
SURF96 R U T  0          29       2.887      0.0848 
SURF96 R U T  0          28        2.86     0.08037 
SURF96 R U T  0          27       2.838     0.07628 
SURF96 R U T  0          26       2.822     0.07266 
SURF96 R U T  0          25       2.814     0.06945 
SURF96 R U T  0          24       2.812     0.06658 
SURF96 R U T  0          23       2.816     0.06399 
SURF96 R U T  0          22       2.828     0.06172 
SURF96 R U T  0          21       2.845     0.05963 
SURF96 R U T  0          20        2.87      0.0578 
SURF96 R U T  0          19       2.898     0.05601 
SURF96 R U T  0          18       2.931     0.05425 
SURF96 R U T  0          17       2.965     0.05245 
SURF96 R U T  0          16           3     0.05053 
SURF96 R U T  0          15       3.034     0.04846 
SURF96 R U T  0          14       3.066     0.04619 
SURF96 R U T  0          13       3.097     0.04375 
SURF96 R U T  0          12       3.123     0.04107 
SURF96 R U T  0          11       3.148     0.03824 
SURF96 R U T  0          10       3.168     0.03522 
SURF96 R U T  0         9.5       3.174     0.03357 
SURF96 R U T  0           9       3.178      0.0319 
SURF96 R U T  0         8.5       3.183     0.03021 
SURF96 R U T  0           8       3.187     0.02852 
SURF96 R U T  0         7.5       3.191     0.02679 
SURF96 R U T  0           7       3.194     0.02505 
SURF96 R U T  0         6.5       3.197     0.02331 
SURF96 R U T  0           6       3.198     0.02153 
SURF96 R U T  0         5.5       3.198     0.01974 
SURF96 R U T  0           5         3.2     0.01796 
SURF96 R U T  0         4.8         3.2     0.01725 
SURF96 R U T  0         4.6       3.201     0.01654 
SURF96 R U T  0         4.4       3.203     0.01584 
SURF96 R U T  0         4.2       3.217     0.01525 
SURF96 R U T  0           4       3.223     0.01458 
