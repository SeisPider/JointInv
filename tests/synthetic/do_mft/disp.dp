SURF96 R U T  0       94.27       3.959      0.5224 
SURF96 R U T  0       89.41       3.948      0.4921 
SURF96 R U T  0        84.2       3.932      0.4611 
SURF96 R U T  0       79.37       3.917      0.4307 
SURF96 R U T  0       74.28       3.896      0.3995 
SURF96 R U T  0        69.4       3.874      0.3686 
SURF96 R U T  0       64.43       3.844       0.337 
SURF96 R U T  0       59.28       3.801      0.3041 
SURF96 R U T  0       54.21       3.744      0.2705 
SURF96 R U T  0       49.14       3.665      0.2356 
SURF96 R U T  0       47.14       3.626      0.2215 
SURF96 R U T  0       45.12       3.579      0.2068 
SURF96 R U T  0          43       3.515      0.1908 
SURF96 R U T  0       41.01       3.453      0.1757 
SURF96 R U T  0       38.98       3.374      0.1598 
SURF96 R U T  0       36.99       3.296      0.1448 
SURF96 R U T  0          35       3.202      0.1295 
SURF96 R U T  0       32.99       3.104       0.115 
SURF96 R U T  0       30.97       3.003      0.1013 
SURF96 R U T  0       29.09       2.923     0.08994 
SURF96 R U T  0       28.12       2.887      0.0848 
SURF96 R U T  0       27.24        2.86     0.08037 
SURF96 R U T  0       26.32       2.838     0.07628 
SURF96 R U T  0       25.43       2.822     0.07266 
SURF96 R U T  0       24.54       2.814     0.06945 
SURF96 R U T  0       23.64       2.812     0.06658 
SURF96 R U T  0       22.73       2.816     0.06399 
SURF96 R U T  0       21.79       2.828     0.06172 
SURF96 R U T  0       20.84       2.845     0.05963 
SURF96 R U T  0       19.85        2.87      0.0578 
SURF96 R U T  0       18.88       2.898     0.05601 
SURF96 R U T  0       17.88       2.931     0.05425 
SURF96 R U T  0       16.89       2.965     0.05245 
SURF96 R U T  0       15.91           3     0.05053 
SURF96 R U T  0       14.92       3.034     0.04846 
SURF96 R U T  0       13.96       3.066     0.04619 
SURF96 R U T  0       12.99       3.097     0.04375 
SURF96 R U T  0       11.99       3.123     0.04107 
SURF96 R U T  0       11.01       3.148     0.03824 
SURF96 R U T  0        9.99       3.168     0.03522 
SURF96 R U T  0       9.526       3.174     0.03357 
SURF96 R U T  0       9.072       3.178      0.0319 
SURF96 R U T  0       8.614       3.183     0.03021 
SURF96 R U T  0       8.121       3.187     0.02852 
SURF96 R U T  0       7.629       3.191     0.02679 
SURF96 R U T  0       7.168       3.194     0.02505 
SURF96 R U T  0       6.695       3.197     0.02331 
SURF96 R U T  0       6.204       3.198     0.02153 
SURF96 R U T  0       5.711       3.198     0.01974 
SURF96 R U T  0       5.247         3.2     0.01796 
SURF96 R U T  0       5.091         3.2     0.01725 
SURF96 R U T  0       4.956       3.201     0.01654 
SURF96 R U T  0       4.862       3.203     0.01584 
SURF96 R U T  0       4.648       3.217     0.01525 
SURF96 R U T  0       4.285       3.223     0.01458 
