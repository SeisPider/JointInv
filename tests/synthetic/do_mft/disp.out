SURF96 R U T  0 95.000    3.4309    1.0000    
SURF96 R C T  0 95.000    6.5741    1.0000    
SURF96 R U T  0 90.000    3.4223    1.0000    
SURF96 R C T  0 90.000    6.2706    1.0000    
SURF96 R U T  0 85.000    3.4131    1.0000    
SURF96 R C T  0 85.000    5.9875    1.0000    
SURF96 R U T  0 80.000    3.4026    1.0000    
SURF96 R C T  0 80.000    5.7341    1.0000    
SURF96 R U T  0 75.000    3.3911    1.0000    
SURF96 R C T  0 75.000    5.4978    1.0000    
SURF96 R U T  0 70.000    3.3783    1.0000    
SURF96 R C T  0 70.000    5.2778    1.0000    
SURF96 R U T  0 65.000    3.3640    1.0000    
SURF96 R C T  0 65.000    5.0723    1.0000    
SURF96 R U T  0 60.000    3.3480    1.0000    
SURF96 R C T  0 60.000    4.8798    1.0000    
SURF96 R U T  0 55.000    3.3297    1.0000    
SURF96 R C T  0 55.000    4.6993    1.0000    
SURF96 R U T  0 50.000    3.3089    1.0000    
SURF96 R C T  0 50.000    4.5281    1.0000    
SURF96 R U T  0 48.000    3.2997    1.0000    
SURF96 R C T  0 48.000    4.4619    1.0000    
SURF96 R U T  0 46.000    3.2901    1.0000    
SURF96 R C T  0 46.000    4.3970    1.0000    
SURF96 R U T  0 44.000    3.2798    1.0000    
SURF96 R C T  0 44.000    4.3332    1.0000    
SURF96 R U T  0 42.000    3.2689    1.0000    
SURF96 R C T  0 42.000    4.2704    1.0000    
SURF96 R U T  0 40.000    3.2573    1.0000    
SURF96 R C T  0 40.000    4.2085    1.0000    
SURF96 R U T  0 38.000    3.2451    1.0000    
SURF96 R C T  0 38.000    4.1474    1.0000    
SURF96 R U T  0 36.000    3.2320    1.0000    
SURF96 R C T  0 36.000    4.0871    1.0000    
SURF96 R U T  0 34.000    3.2182    1.0000    
SURF96 R C T  0 34.000    4.0273    1.0000    
SURF96 R U T  0 32.000    3.2034    1.0000    
SURF96 R C T  0 32.000    3.9679    1.0000    
SURF96 R U T  0 30.000    3.1878    1.0000    
SURF96 R C T  0 30.000    3.9088    1.0000    
SURF96 R U T  0 29.000    3.1796    1.0000    
SURF96 R C T  0 29.000    3.8794    1.0000    
SURF96 R U T  0 28.000    3.1713    1.0000    
SURF96 R C T  0 28.000    3.8500    1.0000    
SURF96 R U T  0 27.000    3.1627    1.0000    
SURF96 R C T  0 27.000    3.8205    1.0000    
SURF96 R U T  0 26.000    3.1538    1.0000    
SURF96 R C T  0 26.000    3.7911    1.0000    
SURF96 R U T  0 25.000    3.1448    1.0000    
SURF96 R C T  0 25.000    3.7616    1.0000    
SURF96 R U T  0 24.000    3.1357    1.0000    
SURF96 R C T  0 24.000    3.7321    1.0000    
SURF96 R U T  0 23.000    3.1263    1.0000    
SURF96 R C T  0 23.000    3.7024    1.0000    
SURF96 R U T  0 22.000    3.1169    1.0000    
SURF96 R C T  0 22.000    3.6727    1.0000    
SURF96 R U T  0 21.000    3.1075    1.0000    
SURF96 R C T  0 21.000    3.6429    1.0000    
SURF96 R U T  0 20.000    3.0981    1.0000    
SURF96 R C T  0 20.000    3.6129    1.0000    
SURF96 R U T  0 19.000    3.0890    1.0000    
SURF96 R C T  0 19.000    3.5829    1.0000    
SURF96 R U T  0 18.000    3.0801    1.0000    
SURF96 R C T  0 18.000    3.5526    1.0000    
SURF96 R U T  0 17.000    3.0719    1.0000    
SURF96 R C T  0 17.000    3.5223    1.0000    
SURF96 R U T  0 16.000    3.0646    1.0000    
SURF96 R C T  0 16.000    3.4919    1.0000    
SURF96 R U T  0 15.000    3.0586    1.0000    
SURF96 R C T  0 15.000    3.4615    1.0000    
SURF96 R U T  0 14.000    3.0545    1.0000    
SURF96 R C T  0 14.000    3.4312    1.0000    
SURF96 R U T  0 13.000    3.0531    1.0000    
SURF96 R C T  0 13.000    3.4011    1.0000    
SURF96 R U T  0 12.000    3.0557    1.0000    
SURF96 R C T  0 12.000    3.3716    1.0000    
SURF96 R U T  0 11.000    3.0638    1.0000    
SURF96 R C T  0 11.000    3.3432    1.0000    
SURF96 R U T  0 10.000    3.0796    1.0000    
SURF96 R C T  0 10.000    3.3165    1.0000    
SURF96 R U T  0 9.5000    3.0913    1.0000    
SURF96 R C T  0 9.5000    3.3041    1.0000    
SURF96 R U T  0 9.0000    3.1060    1.0000    
SURF96 R C T  0 9.0000    3.2926    1.0000    
SURF96 R U T  0 8.5000    3.1243    1.0000    
SURF96 R C T  0 8.5000    3.2822    1.0000    
SURF96 R U T  0 8.0000    3.1463    1.0000    
SURF96 R C T  0 8.0000    3.2732    1.0000    
SURF96 R U T  0 7.5000    3.1723    1.0000    
SURF96 R C T  0 7.5000    3.2658    1.0000    
SURF96 R U T  0 7.0000    3.2017    1.0000    
SURF96 R C T  0 7.0000    3.2605    1.0000    
SURF96 R U T  0 6.5000    3.2326    1.0000    
SURF96 R C T  0 6.5000    3.2573    1.0000    
SURF96 R U T  0 6.0000    3.2603    1.0000    
SURF96 R C T  0 6.0000    3.2566    1.0000    
SURF96 R U T  0 5.5000    3.2743    1.0000    
SURF96 R C T  0 5.5000    3.2576    1.0000    
SURF96 R U T  0 5.0000    3.2528    1.0000    
SURF96 R C T  0 5.0000    3.2585    1.0000    
SURF96 R U T  0 4.8000    3.2257    1.0000    
SURF96 R C T  0 4.8000    3.2578    1.0000    
SURF96 R U T  0 4.6000    3.1826    1.0000    
SURF96 R C T  0 4.6000    3.2556    1.0000    
SURF96 R U T  0 4.4000    3.1189    1.0000    
SURF96 R C T  0 4.4000    3.2509    1.0000    
SURF96 R U T  0 4.2000    3.0294    1.0000    
SURF96 R C T  0 4.2000    3.2425    1.0000    
